Playground: tweaks to <Pagination />

joetaylor-harness opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/harness/canary?pullRequest=240) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/harness/canary?pullRequest=240) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/harness/canary?pullRequest=240) it.</sub>

joetaylor-harness commented:
Minor tweaks to help Sanskar:

- Removed all link structures around `<PaginationEllipsis />` in Playground. The correct way to render is now:
```
<PaginationItem>
  <PaginationEllipsis />
</PaginationItem>
```

- Added new prop `disabled` to `<PaginationPrevious />` and `<PaginationNext />, previously the Previous comp was hardcoded to be lower opacity`:
<img width="462" alt="image" src="https://github.com/user-attachments/assets/6afa0490-8a08-4beb-96fe-65d52e223b91">


- Increased contrast slightly for the active page:
```
<PaginationItem>
  <PaginationLink isActive size="sm_icon" href="#">
     1
  </PaginationLink>
</PaginationItem>
```
<img width="462" alt="image" src="https://github.com/user-attachments/assets/f255abec-a285-4304-9aed-10021eb858c8">


- Page links can now handle large digits:
<img width="438" alt="image" src="https://github.com/user-attachments/assets/cbfebd7f-33ea-42a3-b889-7474f45b76b9">
